.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,31 +1,16 @@
 <!-- DO NOT MODIFY OR DELETE THIS TEMPLATE. IT IS USED IN AUTOMATION. -->
-## Bug
 
-<!-- Search https://github.com/NuGet/Home/issues, and create one if you can't find a suitable issue. -->
-<!-- Paste the full link, like https://github.com/nuget/home/issues/1000. GitHub will render is neatly. -->
-Fixes:
+# Bug
 
-Regression? Last working version:
+<!-- If this is an engineering change or test change only, you do not need an issue. -->
+<!-- Find or create an issue in NuGet/Home and paste the full url. -->
+<!-- At the maintainers discretion, multiple changes may apply to a single issue, but only when the PRs are all created within a short period of time. -->
+Fixes:
 
 ## Description
-<!-- Add details about the fix. Include any information that would help the maintainer review this change effective. -->
 
 ## PR Checklist
 
-- [ ] PR has a meaningful title
-- [ ] PR has a linked issue.
-- [ ] Described changes
-
-- **Tests**
-  - [ ] Automated tests added
-  - **OR**
-  <!-- Describe why you haven't added automation. -->
-  - [ ] Test exception
-  - **OR**
-  - [ ] N/A <!-- Infrastructure, documentation etc. -->
-
-- **Documentation**
-  <!-- Please link the PR/issue if appropriate -->
-  - [ ] Documentation PR or issue filled
-  - **OR**
-  - [ ] N/A
+- [ ] Meaningful title, helpful description and a linked issue
```

Comment:
```suggestion
- [ ] Meaningful title, helpful description and a linked NuGet/Home issue
```

Most repos have issues in the same repo as the code. For contributors not familiar with NuGet.Client, let's make it obvious for them

Reply:
Although I just noticed that you call out NuGet/Home in the comment above the `fixes` line.